packages/oui-file/_variables.less

```diff
@@ -25,6 +25,7 @@
 @oui-file-preview-icon-size: rem-calc(20);
 
 @oui-file-attachment-height: rem-calc(40);
+@oui-file-attachment-max-height: 230px;
```

Comment:
Does it really make sense, even for mobile users ?

Reply:
@FredericEspiau Currently if you have many files you'll get it as a long list and you have to scroll all the way down the page. 

The guidelines are being rewritten so it might change quite soon